assets/stylesheets/common/wizard-admin.scss

```diff
@@ -707,3 +707,26 @@
   width: 80px;
   vertical-align: middle;
 }
+
+.btn-pavilion-pro {
+  background: #3c1c8c;
+  color: white;
```

Comment:
@keegangeorge Likewise this too, so probably ``$pavilion-primary`` and ``$pavilion-secondary``. This makes it easy for us to change it in the future.